spec/api/houdini/nonprofit_spec.rb

```diff
@@ -45,7 +45,7 @@ def h(h = {})
     }
     it 'validates nothing' do
       input = {}
-      xhr :post, '/api/v1/nonprofit', input
+      post '/api/v1/nonprofit', params: input, xhr: true
```

Comment:
xhr and the use of positional params in request specs is deprecated so this corrected that.